querybook/server/datasources/datadoc.py

```diff
@@ -442,9 +442,16 @@ def add_datadoc_editor(
 ):
     with DBSession() as session:
         assert_can_write(doc_id, session=session)
-        editor = logic.create_data_doc_editor(
-            data_doc_id=doc_id, uid=uid, read=read, write=write, commit=False
-        )
+
+        # CREATE & UPDATE FUNCTION ADDED
```

Comment:
Please update this comment to be more meaningful or just remove it, as `create_or_update_data_doc_editor` is quite self-explanatory